assembler/style.css

```diff
@@ -89,6 +89,10 @@ input:not([type=submit]):focus,
     outline: 1px solid currentColor;
 }
 
+.wp-block-jetpack-contact-form-container input[type="radio"] {
```

Comment:
Let's fix this in a less specific way that works for all radio input forms.

Reply:
I have followed the selector's example in the same style file that changes contact form fields. 
https://github.com/Automattic/themes/blob/13e2539b306019250111d660195af4b3813e7fb6/assembler/style.css#L54-L66

I found it safer since I'm not 100% familiar with this particular theme.